sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -216,6 +227,13 @@ class ApiManager {
   }
 
   /**
+   * Gets the user settings JSON from the server.
+   */
+  static getUserSettings() {
+    return ApiManager._xhrAsync('/_settings');
```

Comment:
The point of cacheing here would be to avoid having multiple places that cache. I don't care if it is outside ApiManager, but I think it should be done in one place, and I should be able to call that one place from anywhere.